std/math.d

```diff
@@ -5267,7 +5267,6 @@ int signbit(X)(X x) @nogc @trusted pure nothrow
 ///
 @nogc @safe pure nothrow unittest
 {
-    debug (math) printf("math.signbit.unittest\n");
```

Comment:
This single `debug` statement is almost a decade old, nobody is using it.  Rather than locally import printf here just for this, I deleted it.